cmd/troubleshoot/cli/version.go

```diff
@@ -30,8 +30,9 @@ const VersionFilename = "version.yaml"
 
 func writeVersionFile(path string) error {
 	version := troubleshootv1beta1.SupportBundleVersion{
-		ApiVersion: "troubleshoot.replicated.com/v1beta1",
-		Kind:       "SupportBundle",
+		ApiVersion:    "troubleshoot.replicated.com/v1beta1",
+		Kind:          "SupportBundle",
+		VersionNumber: version.Version(),
```

Comment:
this feels off here. mind putting it under "spec.versionNumber" instead so that it conforms to a k8s yaml convention?

Reply:
done